statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1287,17 +1461,20 @@ def reorder(self, order):
 #-------------------------------------------------------------------------------
 # VAR Diagnostics: Granger-causality, whiteness of residuals, normality, etc.
 
-    def test_causality(self, equation, variables, kind='f', signif=0.05,
-                       verbose=True):
-        """Compute test statistic for null hypothesis of Granger-noncausality,
-        general function to test joint Granger-causality of multiple variables
+    def test_causality(self, causing, kind='f', signif=0.05, verbose=True):
```

Comment:
> i.e. wald/F test that variable in equation is insignificant.

That's right.
The problem I have with the current situation is that the results in section 1 of the [notebook](https://gist.github.com/yogabonito/eedd44be9b8a76de7773a1e43f961d45) I provided are not the same as those in section 2, because in the former there is one variable which is neither contained in `causing` nor in `caused`.

Also, Lutkepohl defines Granger-causality only in the context of a partition of `y` into `causing` (`x` in Lutkepohl) and `caused` (`z` in Lutkepohl) with no remaining variables. Probably this is why also JMulTi is offering to specify only one set of variables and not both `causing` and `caused`.

I think that it would make sense to make a new the method called e.g. `test_granger_causality` such that a new signature is possible and to avoid confusion between `test_causality` and `test_inst_causality` (which I plan to move to var_model.py as added VAR-feature).